src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -15,14 +16,24 @@ namespace NuGet.Services.Metadata.Catalog.Persistence
 {
     public class AzureStorage : Storage
     {
-        private readonly CloudBlobDirectory _directory;
+        private const int _serverTimeoutSeconds = 30;
+        private const int _defaultMaxTimeoutInSeconds = 600;
```

Comment:
How did we decide on these values? They seem like a reasonable start to me, I'm just wondering what the thinking is here.